src/postprocessing/Pass.ts

```diff
@@ -0,0 +1,52 @@
+import { OrthographicCamera, PlaneGeometry, Mesh, Material, Renderer } from 'three'
+
+function Pass() {
+  // if set to true, the pass is processed by the composer
+  this.enabled = true
+
+  // if set to true, the pass indicates to swap read and write buffer after rendering
+  this.needsSwap = true
+
+  // if set to true, the pass clears its buffer before rendering
+  this.clear = false
+
+  // if set to true, the result of the pass is rendered to screen. This is set automatically by EffectComposer.
+  this.renderToScreen = false
+}
+
+Object.assign(Pass.prototype, {
+  setSize: function (/* width, height */) {},
+
+  render: function (/* renderer, writeBuffer, readBuffer, deltaTime, maskActive */) {
+    console.error('THREE.Pass: .render() must be implemented in derived pass.')
+  },
+})
+
+// Helper for passes that need to fill the viewport with a single quad.
+class FullScreenQuad {
+  camera = new OrthographicCamera(-1, 1, 1, -1, 0, 1)
```

Comment:
can we add `private` & `public` to methods & properties? AFAIK props with underscores are always private in this repo, you might wanna check if `camera` & `geometry` are accessed anywhere.

Reply:
Got it. I think given that everything is still in JS, it's pretty painful to go through each and every usage. How about we default to moving underscored fields into private and leave everything else as public? And then over time that can be a refactor step?

Reply:
Yeah, you can also refer to the `threejs` docs, that will also give you an indication, but if you're not sure leave as public.